### ISSUE TITLE
Web admin: animation management

### ISSUE THREAD

**Issue by spicy-catch-268** [YunoHost/issues#2862]
### Description

Following https://github.com/YunoHost/issues/issues/2247 (webadmin - pacman consumes a lot of energy via graphic card on web browser), let's discuss how animations should be managed in the web administration interface.

### Propositions

|Proposition type|Implementation|Consequences|Advantages|Drawbacks|
|----------------------|----------------------|--------------------|-----------------|---------------|
|Current state|"Page transition animations" web admin setting|?|?|?|
|Proposed modification|Add "Animations" web admin setting|If disabled, no animation is present through all web administration interface.|Useful for energy savings, low-end computers, user preferences and peoples with [vestibular disorders](https://www.a11yproject.com/posts/understanding-vestibular-disorders/). Code wide implementation.|User having already configured their OS for reduced motion will also have to modify YunoHost settings.High code modification cost.|
|Proposed modification|Implement media query [prefers-reduced-motion](https://developer.mozilla.org/en-US/docs/Web/CSS/Reference/At-rules/@media/prefers-reduced-motion#User_Preferences)|If enabled, some animations won't be present in web administration interface.|Useful for user preferences and peoples with [vestibular disorders](https://www.a11yproject.com/posts/understanding-vestibular-disorders/). Low code modification cost|Needs to be set in device's settings and thus impact more than just YunoHost. Only applies to CSS based animations.|

### Help needed

- Describe "Page transition animations" web admin setting behavior
- (Propose other solutions)
- (Edit proposed solutions)
- Discuss 🚀 

### Tags for easier GitHub issue search

Those are a list of settings offered by various OS:
Reduced animation, Animation speed, AnimationDurationFactor, Animation Effects, Reduce motion, Remove animations, prefersReducedMotion

**Comment by zamentur** [comment]
Indeed with @Axolotle we discussed this media query during the yunocamp. We agree CSS should be adapted to support that.
